pkg/stream/message.go

```diff
@@ -76,7 +76,7 @@ func (m *Message) Bytes() []byte {
 }
 
 func (m *Message) String() string {
-	return fmt.Sprintf("MESSAGE: connectid %v MessageType %s", m.ConnectID, m.MessageType)
+	return fmt.Sprintf("MESSAGE: ConnectID %v MessageType %s", m.ConnectID, m.MessageType)
```

Comment:
can you use either `connectID` or `ConnectID` i would preffer `connectID`

Reply:
@subpathdev Done. Thanks for the suggestion!